packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -248,6 +248,23 @@ describe('ReactDOMInput', () => {
     }
   });
 
+  it('performs as state change from "" to 0', () => {
```

Comment:
"performs *a state change*

Reply:
Got it in 2ba3fe8e57f3d9c1cfd486d9c2e61a61c23404fb